react-client-app/src/components/ScanReportTbl.jsx

```diff
@@ -336,10 +336,6 @@ const ScanReportTbl = (props) => {
                                     <Link href={"/scanreports/" + item.id + "/assertions/"}><Button variant="green">Assertions</Button></Link>
                                 </Td>
                                 <Td textAlign="center">
-                                    {currentUser &&
-                                        <>
-                                            {currentUser == item.author.username &&
-                                                <>
```

Comment:
Could you please modify the indenting of the inner statements as well.

Reply:
Done